x-pack/plugins/canvas/public/components/es_data_view_select/es_data_view_select.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataView } from '@kbn/data-views-plugin/common';
+import { sortBy } from 'lodash';
+import React, { useRef, useState } from 'react';
+import useEffectOnce from 'react-use/lib/useEffectOnce';
+import { useDataViewsService } from '../../services';
+import {
+  ESDataViewSelect as Component,
+  ESDataViewSelectProps as Props,
+} from './es_data_view_select.component';
+
+type ESDataViewSelectProps = Omit<Props, 'indices' | 'loading'>;
+
+export const ESDataViewSelect: React.FunctionComponent<ESDataViewSelectProps> = (props) => {
+  const { value, onChange } = props;
+
+  const [dataViews, setDataViews] = useState<Array<Pick<DataView, 'id' | 'name' | 'title'>>>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+  const mounted = useRef(true);
+  const { getDataViews } = useDataViewsService();
+
+  useEffectOnce(() => {
+    getDataViews().then((newDataViews) => {
+      console.log({ newDataViews });
```

Comment:
Nit: remove console logs - I thought we had some sort of linting rule that highlighted these - but maybe that's not the case in Canvas?

Reply:
I'm not really sure why the precommit check didn't catch this before letting me commit. I don't think it's a canvas specific issue. Maybe something's wrong with my dev environment?

Reply:
I'm not actually sure if the pre-commit checks get run anymore, I think you might have to run them manually now.